t_reconf/test_reconf_base.py

```diff
@@ -1816,11 +1829,214 @@ class TestNegativeReconf(tester.TempestaTest):
     cookie "__cookie" == "value_1";
     -> default;
 }}
-""",
+            """,
+                deproxy_id="deproxy",
```

Comment:
I suggest to rework all  tests in this class. 
We can use only tls clients and add Tempesta FW config template.
Just add template to class:
```python3
config_template = """
tls_certificate {TEMPESTA_WORKDIR}/tempesta.crt;
tls_certificate_key {TEMPESTA_WORKDIR}/tempesta.key;
tls_match_any_server_name;
frang_limits {{http_strict_host_checking false;}}
"""
and change the test:
```python3
        ...
        tempesta.config.set_defconfig(self.config_template + valid_config)
        tempesta.start()
        tempesta.config.set_defconfig(self.config_template + invalid_config)
        ...
```
```